website/hw/hw5.rst

```diff
@@ -218,7 +229,7 @@ the following:
    note we won’t be grading you on the number of tests your write, but
    on how much coverage they provide.
 
-   Note: Remember that, in Lab 2, you had the option of moving these
+   Note: Remember that, in Homework #2, you had the option of moving these
```

Comment:
Should "orientation" be "point_orientation" on line 236?

Reply:
In HW #2, the function was actually called `orientation` but, in the code provided in this homework, it has been renamed to `point_orientation` (the previous paragraph alludes to this). I've tweaked this paragraph to avoid confusion.